### PR TITLE
add table to ui kit

### PR DESCRIPTION
**Pull request by nshganesh**

### CODE REVIEW

File: src/components/atoms/Table/Components/TableHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { FunctionComponent, ReactNode } from "react";
+
+export type TableHeaderProps = {
+  className?: string,
+  children: ReactNode,
+}
+
+const TableHeader: FunctionComponent<TableHeaderProps> = (props) => {
```
Comment (ankitchouhan1020) [inline src/components/atoms/Table/Components/TableHeader.tsx:8]: `TableHeader = (props: TableHeaderProps)`
FunctionComponent is discouraged. [Why ?](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components)

File: src/components/atoms/Table/Components/TableHeaderCell.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import React, { FunctionComponent, ReactNode } from "react";
+
+import cn from "classnames";
+
+import { IconStore, MI_ICON_LIST } from '../IconStore';
+
+interface SortConfig {
```
Comment (ankitchouhan1020) [inline src/components/atoms/Table/Components/TableHeaderCell.tsx:7]: type convention

File: src/components/atoms/Table/Table.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  static Header = TableHeader;
+  static HeaderCell = TableHeaderCell;
+  static Row = TableRow;
+
+  render() {
+    const { children, className = "", style } = this.props;
+    const classes = cn("tb10Table", className);
+
+    return (
+      <table style={style} className={classes}>
```
Comment (ankitchouhan1020) [inline src/components/atoms/Table/Table.tsx:41]: style undefined handling ?

File: src/components/atoms/Table/Components/TableRow.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
+import React, { Children, isValidElement, cloneElement } from "react";
+
+import cn from "classnames";
+
+type TableRowProps = {
+  className?: string;
+  horzPadding?: string | number;
+  vertPadding?: string | number;
+  id?: string | undefined;
```
Comment (ankitchouhan1020) [inline src/components/atoms/Table/Components/TableRow.tsx:9]: undefined not required, handling for undefined id '' ?

File: src/components/atoms/Table/Components/TableHeaderCell.tsx [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  hideIcon?: boolean;
+  ascending?: boolean;
+  onSortClick?: (e?: React.MouseEvent, ascending?: boolean) => void;
+}
+
+type Props = {
+  className?: string;
+  children: ReactNode;
+  style?: React.CSSProperties;
+  sortConfig?: SortConfig | undefined;
```
Comment (ankitchouhan1020) [inline src/components/atoms/Table/Components/TableHeaderCell.tsx:19]: remove undefined.

**Review by dheerajsingh123 — APPROVED** [review]

**Review by ankitchouhan1020 — APPROVED** [review]

**Pull request merged**